docs/man-pages/condor_submit_dag.rst

```diff
@@ -1,341 +1,358 @@
 *condor_submit_dag*
-=====================
+===================
 
-Manage and queue jobs within a specified DAG for execution on remote
-machines
-:index:`condor_submit_dag<single: condor_submit_dag; HTCondor commands>`\ :index:`condor_submit_dag command`
+Place a node scheduler job to local AP to manage job workflow described as a Direct Acyclic Graph (DAG).
```

Comment:
What's the proper preposition for "placing jobs"?  I think we place jobs "with" the AP.